packages/docusaurus-plugin-content-blog/src/__tests__/authors.test.ts

```diff
@@ -268,11 +268,132 @@ describe('getBlogPostAuthors', () => {
     ]);
   });
 
-  it('can read authors Author', () => {
+  it('can read empty socials', () => {
     expect(
       getBlogPostAuthors({
         frontMatter: {
-          authors: {name: 'Sébastien Lorber', title: 'maintainer'},
+          authors: {
+            name: 'Sébastien Lorber',
+            title: 'maintainer',
+            socials: {},
```

Comment:
We also need tests for undefined/null/absent social key

As you will see your implementation might not do what you expect